tests/integration/suite/daprd/metrics/httpserver_defaultcardinality.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(httpServerDefaultCardinality))
+}
+
+// httpServerDefaultCardinality tests daprd metrics for the HTTP server configured with the default cardinality
+// TODO @ItalyPaleAle [MetricsCardinality] Change default in 1.14 to be based on low cardinality
+type httpServerDefaultCardinality struct {
+	base
+}
+
+func (m *httpServerDefaultCardinality) Setup(t *testing.T) []framework.Option {
+	return m.testSetup(t)
```

Comment:
Please be explicit for the reader and author who will need to update this test when the default changes.

```suggestion
configFile := filepath.Join(t.TempDir(), "config.yaml")
	require.NoError(t, os.WriteFile(configFile, []byte(`
apiVersion: dapr.io/v1alpha1
kind: Configuration
metadata:
  name: lowcardinality
spec:
  metrics:
    http:
      increasedCardinality: null
`), 0o600))

	return m.testSetup(t, procdaprd.WithConfigs(configFile))
```

Reply:
This is a different test then. I thought you wanted to validate the default behavior, which is when there's no Configuration.

Reply:
It should be the same behaviour when the field is set to null, which is the default. It (should be) the same test, we are just being explicit for the reader.

Reply:
There are multiple scenarios being discussed here and one does not exclude the other:
a. low cardinality by setting `increasedCardinality` to False.
b. low cardinality by setting `increasedCardinality` to nil.
c. low cardinality by not setting a config.

If someone changes the default behavior, then (a) will continue to pass while (b) and (c) will fail. For (b) and (c) to behave differently, it would be an odd bug (not necessarily impossible).

I am fine to keep this test as-is for now and have a new PR into master to add more tests but proceed with the release.